src/bundle/burn_validation.rs

```diff
@@ -86,7 +86,7 @@ mod tests {
         let isk = IssuanceAuthorizingKey::from_bytes([1u8; 32]).unwrap();
 
         (
-            AssetBase::derive(&IssuanceValidatingKey::from(&isk), asset_desc),
+            AssetBase::derive(&IssuanceValidatingKey::from(&isk), &asset_desc.into()),
```

Comment:
need to be consistent here and change the signature
```rust
fn get_burn_tuple(asset_desc: &Vec<u8> ..)
```

Reply:
Done. My original motivation was to minimize the code changes in the PR, but I see the logic in being consistent in the code at the cost of more changes in the tests.